stytch/shared/rbac_local.go

```diff
@@ -33,7 +33,7 @@ func PerformAuthorizationCheck(
 		}
 	}
 
-	return stytcherror.NewClientLibraryError("Member is not authorized")
+	return stytcherror.NewSessionAuthorizationError()
```

Comment:
[dust] Can we call this a `NewPermissionError()` or `NewRBACPermissionError()` for parity with other SDKs?

Reply:
[5c737e0](https://github.com/stytchauth/stytch-go/pull/159/commits/5c737e045a12826dcc65a62e80b8f4e6428b88ac)